src/libbson/src/bson/bson.h

```diff
@@ -580,9 +584,17 @@ bson_as_relaxed_extended_json (const bson_t *bson, size_t *length);
 
 
 /* like bson_as_json() but for outermost arrays. */
+BSON_EXPORT (char *) bson_array_as_json (const bson_t *bson, size_t *length);
```

Comment:
Add a note in `bson_array_as_json.rst` suggesting users use the new non-legacy format alternatives. There is a [note in bson_as_json](http://mongoc.org/libbson/current/bson_as_json.html#description) that may be copied and modified:

> This function is superseded by [bson_as_canonical_extended_json()](http://mongoc.org/libbson/current/bson_as_canonical_extended_json.html) and [bson_as_relaxed_extended_json()](http://mongoc.org/libbson/current/bson_as_relaxed_extended_json.html), which use the same [MongoDB Extended JSON format](https://github.com/mongodb/specifications/blob/master/source/extended-json.rst) as all other MongoDB drivers.